adapters/repos/db/aggregator/aggregator.go

```diff
@@ -43,18 +44,14 @@ type Aggregator struct {
 	stopwords              stopwords.StopwordDetector
 	shardVersion           uint16
 	propLengths            *inverted.JsonPropertyLengthTracker
+	propertyIds            *tracker.JsonPropertyIdTracker
 	isFallbackToSearchable inverted.IsFallbackToSearchable
 	tenant                 string
 	nestedCrossRefLimit    int64
+	PropertyName           string
 }
 
-func New(store *lsmkv.Store, params aggregation.Params,
-	getSchema schemaUC.SchemaGetter, classSearcher inverted.ClassSearcher,
-	deletedDocIDs inverted.DeletedDocIDChecker, stopwords stopwords.StopwordDetector,
-	shardVersion uint16, vectorIndex vectorIndex, logger logrus.FieldLogger,
-	propLengths *inverted.JsonPropertyLengthTracker, isFallbackToSearchable inverted.IsFallbackToSearchable,
-	tenant string, nestedCrossRefLimit int64,
-) *Aggregator {
+func New(store *lsmkv.Store, params aggregation.Params, getSchema schemaUC.SchemaGetter, classSearcher inverted.ClassSearcher, deletedDocIDs inverted.DeletedDocIDChecker, stopwords stopwords.StopwordDetector, shardVersion uint16, vectorIndex vectorIndex, logger logrus.FieldLogger, propLengths *inverted.JsonPropertyLengthTracker, isFallbackToSearchable inverted.IsFallbackToSearchable, tenant string, propertyIds *tracker.JsonPropertyIdTracker, nestedCrossRefLimit int64) *Aggregator {
```

Comment:
Can we bring back the old formatting here? The line brakes here where introduced intentionally so that when somebody works using side by side with two tabs, he then is able to read method's signature without having to scroll to the end of the line to check what are the aguments.